packages/babel-plugin-remove-graphql-queries/src/index.ts

```diff
@@ -275,8 +275,7 @@ export default function ({ types: t }): PluginObj {
           JSXIdentifier(path2: NodePath<JSXIdentifier>): void {
             if (
               (process.env.NODE_ENV === `test` ||
-                state.opts.stage === `develop-html` ||
-                state.opts.stage === `build-html`) &&
+                state.opts.stage === `develop-html`) &&
```

Comment:
breaking change #1 - babel plugin no longer inline static queries in `build-html`

Reply:
Just curious, why only `build-html`? It is not immediately clear.

Reply:
For the most part because I don't "care" about `develop-html` in this PR (other than it still working heh). 

I remove static queries results inlining from Prod SSR bundle so it doesn't impact SSR bundle compilation hash so we don't hit case of SSR bundle hash changing invalidating ALL the pages (so we can invalidate only pages that actually are affected by static query result changes).